src/Native/Silk.NET.MoltenVK.Native/Silk.NET.MoltenVK.Native.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+    <Import Project="..\..\..\build\props\common.props" />
```

Comment:
Thank you for the binaries, however we are trying to move away from having binaries from an unknown (potentially untrusted) source in the repo. Recently, we have moved to defining all of our build scripts in this directory: https://github.com/dotnet/Silk.NET/tree/main/build/nuke/Native with automation here: https://github.com/dotnet/Silk.NET/tree/main/.github/workflows

I think we would want similar for MoltenVK as well. If you're happy to add a build script and a workflow, that would be amazing! But otherwise, let's just remove this import which will prevent these binaries from being shipped until someone can add trusted build scripts and CI workflows. It's nothing personal, we just try to always have binaries built from CI for security reasons.

Reply:
Yes, I understand your concerns about binaries.
I think I can write build script to load MoltenVK-all.tar from official MoltenVK repo, unpack and take this .xcframework. Or even pick up only the required libraries, if you think it's better that way. 
But I cannot guarantee how long The Khronos Group will continue to maintain this structure.

Reply:
I added the MoltenVK repository as a submodule and added the build script. But there are several difficulties:
1) MoltenVK is built in Xcode, so I only build if OperatingSystem.IsMacOS()
2) Building MoltenVK requires cmake, python3 and ninja, as well as brew to install them (https://github.com/KhronosGroup/MoltenVK#install). I'm not sure if I need to add these installation steps or not?

Reply:
I looked into the capabilities of GitHub Actions and I think the new version should work

Reply:
Looks good, thanks!